### PR TITLE
Feature: adding index-based getters for up and down quark running masses

### LINKED ISSUE #498: displayUpQuarkPoleMass etc
`FlexibleDecay` in `src/decays/decay_functions.cpp` introduced `sm_down_quark_running_masses` function (and analogue one for up-type quarks). Could I get for uniformity a set of function similar to #479 but for quarks?

### PR DESCRIPTION
**Pull request by Expander**
fixes #498

### CODE REVIEW

File: src/decays/decay_functions.cpp [hunk 1]
```diff
@@ -305,24 +305,15 @@ unsigned int number_of_active_flavours(softsusy::QedQcd const& qedqcd, double m)
 
 double sm_up_quark_masses(softsusy::QedQcd const& qedqcd, int n)
 {
-   switch(n) {
-      case 0: return qedqcd.displayMass(softsusy::mUp);
-      case 1: return qedqcd.displayMass(softsusy::mCharm);
-      default:
-         throw std::runtime_error("Unknown quark mass");
+   if (n == 2) {
+      throw std::runtime_error("sm_up_quark_masses: top quark mass cannot be used");
```
Comment (wkotlarski) [inline src/decays/decay_functions.cpp:309]: Why excluded a top quark? We also need it.

Comment (Expander) [reply src/decays/decay_functions.cpp:309]: It was excluded before. :) If we need it, I'll modify this PR.

Comment (Expander) [reply src/decays/decay_functions.cpp]: Done in 3c89d4163

Comment (wkotlarski) [reply src/decays/decay_functions.cpp]: Oh, sorry. I only vague remember why I did it like that :cry:

**Review by wkotlarski — APPROVED** [review]

File: src/decays/decay_functions.cpp (hunk 1 above)
Comment (Expander) [reply src/decays/decay_functions.cpp]: Just be sure: We must include the top quark, right? I.e. the current status of this PR is correct, right? :)

**Pull request merged**